ConsoleApp/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿var text1 = "Hello World123/.,m!";
+var text2 = "Hello Danylo!";
+System.Console.WriteLine(text1, text2);
+
+var res1 = Compare(text1, text2);
+System.Console.WriteLine(res1);
+
+var res2 = Sort(text1);
+System.Console.WriteLine(res2);
+
+var res3 = Analyze(text1);
+System.Console.WriteLine($"Digit: {res3.Item1} Letter: {res3.Item2} Number: {res3.Item3}");
+
+var text3 = "Hello World!";
+var text4 = "Hello Danylo!";
+var resault =  Duplicate(text3, text4);
+System.Console.WriteLine($"Duplicate characters: {resault}");
+
+
+bool Compare(string t1, string t2)
+{
+    for (int i = 0; i < t1.Length; i++)
```

Comment:
it won't work if, for example, I pass a substring of string, e.g. `hell` and `hello`